test/utils/Constants.sol

```diff
@@ -14,8 +14,8 @@ library Constants {
     uint160 constant MAX_UINT160 = type(uint160).max;
 
     address constant ADDRESS_ZERO = address(0);
-    address constant EMPTY_HOOKS = address(0xf000000000000000000000000000000000000000);
-    address constant MOCK_HOOKS = address(0xfF00000000000000000000000000000000000000);
+    address constant EMPTY_HOOKS = address(0x000000000000000000000000000000000000f000);
```

Comment:
ah this should be `3c00` not `f000` to be equivalent.
side note the name `EMPTY_HOOKS` doesnt make sense but thts a pre-existing issue that we will fix sometime soon